application/factory.py

```diff
@@ -48,6 +49,8 @@
     first_bullet,
 )
 
+csrf = CSRFProtect()
```

Comment:
I think this might sit more nicely in `application/__init__.py`, where some of the other central app resources sit - what do you think?